lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/services/FileWatchService.java

```diff
@@ -68,7 +68,10 @@ public void addFileAlterationObserver(LibertyWorkspace workspace, List<String> w
 
     private FileAlterationObserver getFileAlterationObserver(final String parentPath, LibertyWorkspace workspace) {
         IOFileFilter notFileFilter = FileFilterUtils.notFileFilter(
-                new SuffixFileFilter(Arrays.asList(".class", ".lst", ".txt"), IOCase.INSENSITIVE));
+                new SuffixFileFilter(Arrays.asList(".class", ".lst", ".txt",".log",".manager",".libertyls",".sLock"),
```

Comment:
I like this!

I think we can add some more file extensions here: .jar, .war, .ear

Also some dirs: libs, tmp, classes, generated-sources, generated-test-sources, invoker-reports, it, maven-status, surefire-reports, test-classes

Do you see a down side to adding these?

Reply:
That's fine. We don't care about anything in these subfolders.